src/formio.service.ts

```diff
@@ -36,13 +36,22 @@ export class FormioService {
   loadForm(options?: any): Observable<FormioForm> {
     return this.requestWrapper(() => this.formio.loadForm(options));
   }
-  loadSubmission(): Observable<{}> {
-    return this.requestWrapper(() => this.formio.loadSubmission());
+  loadForms(query: any): Observable<FormioForm> {
+    return this.requestWrapper(() => this.formio.loadForms(query));
+  }
+  loadSubmission(query?: any, opts?: any): Observable<{}> {
+    return this.requestWrapper(() => this.formio.loadSubmission(query, opts));
+  }
+  userPermissions(user: any, form: any, submission: any): Observable<{}> {
+    return this.requestWrapper(() => this.formio.userPermissions(user, form, submission));
+  }
+  deleteSubmission(opts?: any): Observable<{}> {
+    return this.requestWrapper(() => this.formio.deleteSubmission(opts));
   }
   saveSubmission(submission: {}): Observable<{}> {
     return this.requestWrapper(() => this.formio.saveSubmission(submission));
   }
-  loadSubmissions(): Observable<{}> {
+  loadSubmissions(query?: any, opts?: any): Observable<{}> {
     return this.requestWrapper(() => this.formio.loadSubmissions());
```

Comment:
Don't you need to pass along the query and opts to ```loadSubmissions``` here?

Reply:
That's my bad. Fixed it.

Reply:
That's my bad. Fixed it.